pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -983,6 +997,12 @@ type CephBlockPoolStatus struct {
 	Conditions         []Condition `json:"conditions,omitempty"`
 }
 
+// PoolCephxStatus represents the cephX key rotation status for cephBlockPool
+type PoolCephxStatus struct {
```

Comment:
Looking ahead, this will proably get reused for CephFilesystem in some way, so a more generalized name will be good

```suggestion
// PeerTokenCephxStatus represents the cephX key rotation status for cephBlockPool
type PeerTokenCephxStatus struct {
```